Neos.ContentGraph.PostgreSQLAdapter/src/Domain/Projection/HypergraphProjection.php

```diff
@@ -98,10 +96,10 @@ public function __construct(
     public function setUp(): void
     {
         $this->setupTables();
-        $this->checkpointStorage->setup();
+        $this->checkpointStorage->setUp();
```

Comment:
it seems you decided against

> rename setUp() to setup() for consistency 

from your original pr? https://github.com/neos/neos-development-collection/pull/4843

but either way just a minor thing. Maybe lets create a followup to clean this up and not have it intertwined here.

Reply:
Edit: it seems by the look at the `AssetUsageRepository::setUp` that you decided for the `setUp` now. Fine with me as well :D Happy to see that its consistent in one or the other way.

Fun fact php doesnt care how we call the method so one yould still write `setup` 😂

Reply:
Yes, good catch, I forgot to mention that again.
I realized that we use `setUp` more often in the core and it makes sense because "setup" does not exist as a verb..
Unfortunately we went for `setup` in the eventstore packages but I don't think it's worth a new release to change that